rust/frontend/src/api.rs

```diff
@@ -0,0 +1,55 @@
+use chroma_config::Configurable;
+use chroma_error::ChromaError;
+use chroma_sysdb::sysdb;
+use chroma_types::CreateDatabaseResponse;
+
+use crate::config::FrontEndConfig;
+
+#[async_trait::async_trait]
+pub trait ServerApi {
+    async fn create_database(
+        &mut self,
+        request: chroma_types::CreateDatabaseRequest,
+    ) -> chroma_types::CreateDatabaseResponse;
+}
+
+#[derive(Clone)]
```

Comment:
I don't think this should go in /frontend. this _is_ chroma. it should go in /chroma